salt/buildbot/init.sls

```diff
@@ -70,10 +69,12 @@ update-master:
   cmd.run:
     - runas: buildbot
     - cwd: /srv/buildbot
-    - name: make update-master
+    - name: make update-master PYTHON_VERSION=3.13
```

Comment:
Is there a handy way to set a "buildbot_python_version" variable somewhere and use it here and in the package names above so that there's just one place to update?